build/transpile.js

```diff
@@ -889,6 +892,9 @@ class Transpiler {
     // exchange capabilities ordering
 
     sortExchangeCapabilities (code) {
+        const baseExchange = this.getBaseClass ()
+        const defaultDescribe = baseExchange.describe ();
```

Comment:
hum, this assumes the code was transpiled first to JS? Usually that requirement was not needed

Reply:
no, this method was in place before transpilation happens (sorts the `has` dict) and after that starts transpilation.
if you look this line where this method is called: 
https://github.com/ccxt/ccxt/blob/f22d9b2f87a9739cd156676a08fa4090ea2f0809/build/transpile.js#L1284

down below 2 lines then the .ts file is overwritten, and transpilation starts then with the "sorted" code.

Reply:
I avoided to include all changes that happen after running transpiler, so if you want to see the changes it does to any exchange, you might run the transpiler and see diff for `binance` or any exchange